packages/runtime/container-runtime/src/gc/gcEarlyAdoption.md

```diff
@@ -89,12 +89,14 @@ Note: The Summarizer client will _never_ throw on usage or load of a Tombstoned
 
 ## Enabling Sweep
 
-The following configuration is required for Sweep to be enabled for a given document:
+To enable Sweep for the first time, set this GC Option:
 
--   GC Option `gcSweepGeneration` must be set, and the persisted value must match the current value in the code
--   Each of these two Config Settings must be set to `true` in the session:
-    -   `Fluid.GarbageCollection.Test.SweepDataStores`
-    -   `Fluid.GarbageCollection.Test.SweepAttachmentBlobs`
+```ts
+gcSweepGeneration: 0;
+```
+
+The exception would be if you ever bumped `gcTombstoneGeneration` - in that case, set `gcSweepGeneration` to `1` or higher.
```

Comment:
This is confusing. I would explain what the number 0 means here and when to set it to a higher value.

Then follow that up with a note that talks about its interaction with `gcTombstoneGeneration` and how there is a special case when `gcTombstoneGeneration` is > 0 (?).

Reply:
Yeah, in writing these docs I realized how random / non-intuitive this all is.  I'll try to reword it.